pkg/utils/report/labels.go

```diff
@@ -66,6 +67,17 @@ func PolicyLabel(policy kyvernov1.PolicyInterface) string {
 	return PolicyLabelPrefix(policy) + policy.GetName()
 }
 
+func CleanupKyvernoLabels(obj metav1.Object) {
+	labels := obj.GetLabels()
+	for key := range labels {
+		if strings.Contains(key, LabelDomain) {
```

Comment:
can we use `strings.HasPrefix` ?

Reply:
The issue is that we have different prefix. There is `audit.kyverno.io`, `cpol.kyverno.io` and `pol.kyverno.io`

Reply:
`audit.kyverno.io`, `cpol.kyverno.io` and `pol.kyverno.io` should probably not be preserved across reconcile loops.

Reply:
Sorry, i read it wrong.